cogs/checkip.py

```diff
@@ -1,12 +1,14 @@
 from asyncio import wait_for
+
+import discord
 from discord import slash_command, option, Embed
-from discord.ext import commands
 from discord.ext.commands import Bot
+
 from utilities.data import Colors, get_server_status
 from utilities.utility import check_ip
```

Comment:
Please group the respective imports together. Why the weird newlines between imports?

Reply:
To seperate third party imports (discord) from local imports (utilities.data) and built-in imports (asyncio)